website/config.toml

```diff
@@ -0,0 +1,114 @@
+baseURL = "/"
+title = "Cortex"
+
+enableRobotsTXT = true
+
+# Hugo allows theme composition (and inheritance). The precedence is from left to right.
+theme = ["docsy"]
+
+# Will give values to .Lastmod etc.
+enableGitInfo = true
+
+# Language settings
+contentDir = "content/en"
+defaultContentLanguage = "en"
+defaultContentLanguageInSubdir = false
+# Useful when translating.
+enableMissingTranslationPlaceholders = true
+
+disableKinds = ["taxonomy", "taxonomyTerm"]
+
+# Highlighting config
+pygmentsCodeFences = true
+pygmentsUseClasses = false
+# Use the new Chroma Go highlighter in Hugo.
+pygmentsUseClassic = false
+#pygmentsOptions = "linenos=table"
+# See https://help.farbox.com/pygments.html
+pygmentsStyle = "tango"
+
+# Configure how URLs look like per section.
+[permalinks]
+blog = "/:section/:year/:month/:day/:slug/"
+
+## Configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
+[blackfriday]
+plainIDAnchors = true
+hrefTargetBlank = true
```

Comment:
>> Links in https://cortexmetrics.io/docs/ open in a new tab, which looks weird for internal links like the ones in the Documentation section (https://cortexmetrics.io/docs/#documentation)

> I couldn't find how to control this. It seems that Hugo is making all doc content link, blank ones.

It's this one. You should switch it to `false`.

Reply:
Giving a try :)

Reply:
Yes that works.